plugins/services/src/js/components/modals/AppLockedMessage.js

```diff
@@ -17,10 +19,10 @@ const AppLockedMessage = function({ service }) {
   }
 
   return (
-    <h4 className="text-align-center text-danger flush-top">
+    <Trans render="h4" className="text-align-center text-danger flush-top">
       {itemType} is currently locked by one or more deployments. Press the
```

Comment:
I think that is what we are doing here. The service type will be translated with non-JSX resources and replaced into this text. But the rest of this will be translated as one piece.